packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerService.ts

```diff
@@ -177,12 +184,42 @@ export class OpenId4VcIssuerService {
     }
 
     const vcIssuer = this.getVcIssuer(agentContext, issuer)
+    const issuerMetadata = this.getIssuerMetadata(agentContext, issuer)
+    const holderBinding = await this.getHolderBindingFromRequest(credentialRequest as OpenId4VciCredentialRequest)
```

Comment:
At this stage this binding hasn't been verified yet, I was thinking we do the check before we call `mapper` , but stilli in the callback we pass to the sphereon lib. That way we know the request etc.. is already verified. But maybe there's something preventing that?

Reply:
In the `getCredentialDataSupplier` we have access to the `issuanceSessionRecord` right? 

So everything you now moved out of the `getCredentialDataSupplier` ca be in the `getCredentialDataSupplier`?